### ISSUE TITLE
Can not open Worlds （Win10 uwp 1.7）

### ISSUE THREAD

**Issue by fx-k** [Podshot/MCEdit-Unified#964]
Click the urls to view the error：
http://qiniu.fxit.top/o_1ctudfotq1miq1iu2jjj12fpn06a.jpeg

**Comment by Podshot** [comment]
What version of MCEdit-Unified are you using? For future reference, please directly upload screenshot directly to your report/comment

**Comment by fx-k** [comment]
![222](https://user-images.githubusercontent.com/17313185/50320907-62168b80-0509-11e9-8096-1da38b81eb4a.PNG)
paste problems

**Comment by Podshot** [comment]
Can you give us a screenshot of the console window with this error?

**Comment by Teddybear335** [comment]
![erroeMC](https://user-images.githubusercontent.com/42890756/55282596-43653380-5347-11e9-816b-c594297a54df.png)
 Version: 1.5.6.0
Win. 10

**Comment by gentlegiantJGC** [comment]
This has nothing to do with this bug report. See the test releases for Bedrock support
https://github.com/Podshot/MCEdit-Unified-Preview/releases/tag/MCEdit-Unified-1.6.0.53-testing

**Comment by michael-ts** [comment]
@gentlegiantJGC I'm having this same issue and am trying to load a Bedrock world.  Can you explain what the problem is?  My understanding is that MCEdit-Unified should be able to open bedrock worlds.  Is version 1.6.0.53 required to support this?  If so, I just went through the process of installing 1.5.6.0 MCEdit.v1.5.6.0.Lin.Universal.run - what do I need to do now?

(*) Technically I got a slightly different error when trying to load level.dat: 
`LoadingError: (('Multiple errors encountered', NBTFormatError('Not an NBT file with a root TAG_Compound (file starts with "   \x08" (0x00000008)',), IOError('MCJavaLevel attempted for smaller than 64 blocks cubed',)), <traceback object at 0x7fc240a0f908>)`

Update: I downloaded the [1.6.0.0](https://github.com/Podshot/MCEdit-Unified/tags) source but it still won't load.  Where can I find the 1.6.0.53 *source*?

**Comment by michael-ts** [comment]
I don't know if it makes a difference or not, but I appear to be running Minecraft Bedrock (Android) 1.11.4.2,  but MCEdit says it is for "Minecraft 1.8-1.12.2 | PE 1.6".  I'm a bit confused on the naming, is 1.11.4 included in the first range or does PE refer not only to Pocket Edition but also the Bedrock Edition?

**Comment by gentlegiantJGC** [comment]
You need the latest 1.6 to open Bedrock worlds. PE became Bedrock and the name in bedrock has stayed the same.
You can get 1.6 from the website under more versions in the bottom right of the top block. I think the link says dev versions or test release versions or something like that

**Comment by michael-ts** [comment]
@gentlegiantJGC Thank you, but I have tried that and am not finding 1.6.0.53 for Linux.  I assume by "the website" you mean https://www.mcedit-unified.net/?  The "bottom right of the top block" meaning under "Linux Beta"?  I see " Click here to view all versions".  When I click on that it changes to "Pre-Anvil versions can be found at this page."  That links leads to https://github.com/Khroki/mcedit-old/releases?after=0.1.6build315 which is old old old, not what I am looking for at all.  Back on the previous page I also see "Click here for test releases (Windows)" but I don't have Windows so that doesn't help me.

If I go all the way to the bottom of the [website](https://www.mcedit-unified.net/) I see "More Releases", this takes me [here](https://github.com/Podshot/MCEdit-Unified/releases) where I can see and download 1.6.0.0... but this still won't open my world (error in previous comment).  Where can I get the *latest* 1.6 at?

Thanks!

**Comment by michael-ts** [comment]
@gentlegiantJGC I noticed that the 1.6.0.0 release at the above link was said to be 4 commits behind the master - I downloaded *that* too but when I run it, it *still* says 1.6.0.0 and won't open my world.

**Comment by Podshot** [comment]
@michael-ts Are you still getting the same error that you posted previously?

**Comment by michael-ts** [comment]
@Podshot yes.when i download and run mcedit.sh from the master I get the error:

> I don't know how to open /home/michael/.minecraft/minecraftWorlds/Test/level.dat:
>
> AttributeError("'NoneType' object has no attribute 'Options'",) 

I assume level.dat is the correct file to open, as it's the only one that shows up as openable.

This was a freshly created world in Minecraft Bedrock (Android) 1.11.4.2 running on Linux under the Minecraft Bedrock Launcher.

**Comment by michael-ts** [comment]
@Podshot Please let me know if there is additional information I can provide.
